docs/features/workspaces.md

```diff
@@ -52,6 +52,16 @@ Within the repositories dashboard, you can view the following metrics, which are
 - **Issues**: This includes the total opened and closed issues for the repositories included in your workspace, as well as the velocity of issues being closed.
 - **Engagement**: This includes the total stars, forks, and activity ratio for the repositories included in your workspace.
 
+#### Activity Dashboard
+
+The Activity Dashboard provides a centralized view of recent activity across repositories, including detailed tables for Pull Requests (PRs) and Issues. These tables offer insights into ongoing contributions, bug fixes, feature requests, and more. 
+
+##### Pull Requests Table
+The Pull Requests table lists recent PRs across the repositories in your Workspace, providing details to track progress and contributions and includes information on the current status of the PR, the number of files changed, the number of additions and deletions, and timestamps for updates, creation, closure, and merging.
+
+##### Issues Table
+The Issues table displays recent issues from the repositories in your workspace, helping you to identify and track bugs, feature requests, and ongoing discussions. The issues table gives you insight into the current status of the issue, the number of reactions, and timestamps for updates, creation, and closure.
```

Comment:
Let's:
- add a space after the heading.
- avoid the duplication of "The issues table" by changing it to "It" in the 2nd sentence.

```suggestion
##### Issues Table

The Issues table displays recent issues from the repositories in your workspace, helping you identify and track bugs, feature requests, and ongoing discussions. It gives you insight into the current status of the issue, the number of reactions, and timestamps for updates, creation, and closure.
```